htdocs/expedition/card.php

```diff
@@ -63,7 +63,7 @@
 }
 
 // Load translation files required by the page
-$langs->loadLangs(array("sendings", "companies", "bills", 'deliveries', 'orders', 'stocks', 'other', 'propal'));
+$langs->loadLangs(array("sendings", "companies", "bills", 'deliveries', 'orders', 'stocks', 'other', 'propal', 'errors'));
```

Comment:
Errors are "uncommon case" . We want the file error.log that will become larger and larger being loaded only if we need it instead of at each page call. Can you load it instead just before the lang->trans that need error sentence ?